app/javascript/controllers/conditional_question_controller.js

```diff
@@ -6,13 +6,25 @@ export default class extends Controller {
   }
 
   displayConditional() {
+    switch(this.element.type) {
+      case "number":
+        this.displayConditionalNumeric()
+      case "radio":
+        this.displayConditionalRadio()
+      default:
+        console.log("Not yet implemented for " + this.element.type)
+        break;
+    }
```

Comment:
This will likely grow, I take it there's no option to pass a method through `data-info` 😅 Have you had any ideas of how this could be structured differently?